File: cpp/examples/mesh_map_registration.cpp
#include "registration_common.hpp"

int main() {
    auto set = examples::robot_description_fleet();
    set.datavizs.front().add_3d_mesh(
        "/map_3d_mesh",
        "map",
        {{"max_triangles", 60000}, {"use_vertex_colors", true}});
    return examples::register_or_fail(set);
}

File: rust/examples/mesh_map_registration.rs
mod common;

fn main() {
    let mut set = common::robot_description_fleet();
    set.datavizs[0].add_3d_mesh(
        "/map_3d_mesh",
        "map",
        Some(common::options(&[
            ("max_triangles", serde_json::json!(60000)),
            ("use_vertex_colors", serde_json::json!(true)),
        ])),
    );
    common::register_or_fail(&mut set);
}
